representable.gemspec

```diff
@@ -27,7 +27,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rake"
   spec.add_development_dependency "test_xml", "0.1.6"
   spec.add_development_dependency "minitest"
-  spec.add_development_dependency "virtus"
+  spec.add_development_dependency "dry-types", '~> 0.9'
```

Comment:
@apotonick  dry-types > 0.9.0 now [requires a minimum Ruby version of 2.1.0](https://github.com/dry-rb/dry-types/commit/3f986ba8509e0119445ded570e5d3e5c66359df9) - I only tested with 0.9, but we could probably allow a lesser version here, if support for coercion is important to Ruby 2.0 users. 

Ruby 1.9.3 users might still be out of luck though (need to check with @flash-gordon/@solnic, but I don't think dry-types was ever tested on that Ruby).